source/win-main/store.ts

```diff
@@ -230,6 +230,10 @@ export interface ZettlrState {
    * This variable stores search results from the global search
    */
   searchResults: SearchResultWrapper[]
+  /**
+   * Cached texts of the headings
+   */
+  headingCache: { [key: string]: string }
```

Comment:
Since JavaScript now has support for Maps, I think we should be using those. Also -- according to the internet (tm) -- Maps are faster: https://bretcameron.medium.com/how-javascript-maps-can-make-your-code-faster-90f56bf61d9d

Reply:
I'm using Maps frequently in Vue components, feel free to search for `new Map()` in the codebase